pkg/gateway/servicemap.go

```diff
@@ -41,3 +43,24 @@ type Endpoint struct {
 	IP   string
 	Port int
 }
+
+type Aliases struct {
+	Data map[string]string
```

Comment:
Keeping w/ the `ServiceMap` naming convention, could we call this `AliasMap` and simply alias the type to `map[string]string`?

```
type AliasMap map[string]string
```